### PR TITLE
Slovenian: Use Cases

### PR DESCRIPTION
**Pull request by wackerow**
## Description
Updates Slovenian (`sl`) Use Cases content with latest from Crowdin.

### CODE REVIEW

File: src/content/translations/sl/dao/index.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+## Kako delujejo DAOs? {#how-daos-work}
+
+Podlaga za DAO je pametna pogodba. Ta pogodba določa pravila organizacije in hrani sredstva skupine. Ko je pogodba aktivna na Ethereumu, nihče ne more spremeniti pravil, razen z glasovanjem. Če kdorkoli poskusi narediti nekaj kar ni določeno v pravilih in logiki programske kode, ta poskus ne bo uspel. In ker je blagajna prav tako opredeljena s pametno pogodbo to pomeni, da nihče ne more porabiti sredstev brez soglasja skupine. To pomeni, da DAOs ne potrebujejo osrednje oblasti. Namesto tega skupina sprejema odločitve kot celota in po uspešnem glasovanju so plačila odobrena samodejno.
+
+To je mogoče, saj so pametne pogodbe odporne proti posegom, ko so enkrat aktivne na Ethereumu. Ne morete kar urejati kode (DAO pravil) brez, da bi kdo opazil, saj je vse javno dostopno.
+
+<DocLink to="/developers/docs/smart-contracts/" title="Več o pametnih pogodbah" />
```
Comment (minimalsm) [inline src/content/translations/sl/dao/index.md:78]:
```suggestion
<DocLink to="/developers/docs/smart-contracts/">
  Več o pametnih pogodbah
</DocLink>
```

File: src/content/translations/sl/defi/index.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+- [DeFi Prime](https://defiprime.com/)
+
+### DeFi članki {#defi-articles}
+
+- [Kaj so decentralizirane finance?](https://docs.ethhub.io/built-on-ethereum/open-finance/what-is-open-finance/) – _ETHHub, redno posodobljen_
+- [Začetniški vodnik po DeFi](https://blog.coinbase.com/a-beginners-guide-to-decentralized-finance-defi-574c68ff43c4) – _Sid Coelho-Prabhu, 6. januar 2020_
+
+### Videoposnetki {#videos}
+
+- [Finematics - decentralized finance education](https://finematics.com/) – _Videos on DeFi_
```
Comment (minimalsm) [inline src/content/translations/sl/defi/index.md:318]: @lukassim any reason we wouldn't have translated this resource but translated the rest?

**Review by minimalsm — CHANGES_REQUESTED** [review]
not finished review just saving comments

File: src/content/translations/sl/defi/index.md (hunk 2 above)
Comment (lukassim) [reply src/content/translations/sl/defi/index.md:318]: No, this should be translated.
Looking at Crowdin, the translator translated this, but the reviewer changed it back to English.

Comment (lukassim) [reply src/content/translations/sl/defi/index.md:318]: @minimalsm Here's the translation:
[Finematics - izobraževanje na temo decentraliziranih financ](https://finematics.com/) – _Videoposnetki o decentraliziranih financah_

File: src/content/translations/sl/contributing/translation-program/translation-guide/index.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+Če ste novi v prevajalskem programu in se še obotavljate, so tukaj FAQ - pogosto zastavljena vprašanja, ki vam lahko pomagajo pri začetku. Uporabite ta vodnik, da najdete odgovore do najpogostejših poizvedb.
+
+## Kako prevedem te nize z `<HTML tags>`? {#tags}
+
+Ni vsak niz napisan v čisti besedni obliki. Obstajajo nekateri nizi, ki so sestavljeni iz mešanih skript, kot so HTML tagi (`<0>`, `</0>`). Te se po navadi uporabljajo za hiperpovezave ali alternativno oblikovanje v sredini stavka.
+
+- Prevedite besedilo znotraj tagov, ampak ne samih tagov. Karkoli v `<` in `>` ne sme biti prevedeno ali odstranjeno.
+- Da ohranite niz varen, priporočamo, da kliknete "Copy Source" gumb spodaj levo. To bo kopiralo izvorni niz in ga prilepilo v besedilno polje. To vam omogoči razjasniti, kje so tagi in vam pomaga pri izogibanju napakam.
+
+![Crowdin vmesnik z izpostavljenim gumbom kopiraj vir](./html-tag-strings.png)
```
Comment (minimalsm) [inline src/content/translations/sl/contributing/translation-program/translation-guide/index.md:19]:
```suggestion
![Crowdin vmesnik z izpostavljenim gumbom kopiraj vir](../../../../../contributing/translation-program/translation-guide/html-tag-strings.png)
```

File: src/content/translations/sl/contributing/translation-program/translation-guide/index.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+Lahko premaknete položaj tagov znotraj niza, da bo bolje zvenel v vašem jeziku – le prepričajte se, da ste premaknili celoten tag.
+
+## Kje živijo nizi? {#strings}
+
+Pogosto le izvorni niz morda ne bo dovolj, da bi lahko zagotovili natančen prevod.
+
+- Oglejte si posnetke zaslona in kontekst za več informacij. V sekciji izvornega niza boste našli pripet posnetek zaslona, ki vam bo pokazal kako niz uporabljamo v kontekstu.
+- Če še vedno niste prepričani postavite vprašanje v sekciji za komentarje. [Niste prepričani kako podati komentar?](#comment)
+
+![Prikaz načina zagotovitve konteksta za niz s posnetkom zaslona](./source-string.png)
```
Comment (minimalsm) [inline src/content/translations/sl/contributing/translation-program/translation-guide/index.md:30]:
```suggestion
![Prikaz načina zagotovitve konteksta za niz s posnetkom zaslona](../../../../../contributing/translation-program/translation-guide/source-string.png)
```

File: src/content/translations/sl/contributing/translation-program/translation-guide/index.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+![Primer posnetka zaslona za kontekst](./source-string-2.png)
```
Comment (minimalsm) [inline src/content/translations/sl/contributing/translation-program/translation-guide/index.md:32]:
```suggestion
![Primer posnetka zaslona za kontekst](../../../../../contributing/translation-program/translation-guide/source-string-2.png)
```

File: src/content/translations/sl/contributing/translation-program/translation-guide/index.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+## Kako lahko podam komentar ali postavim vprašanje? Rad bi označil težavo ali napako... {#comment}
+
+Če želite označiti določen niz, ki potrebuje pregled, lahko oddate komentar.
+
+- Kliknite drugi gumb v vrstici desno zgoraj. Skrit zavihek se bo pojavil na vaši desni. Podajte nov komentar in kliknite okence "težava" na dnu. Vrsto težave lahko opredelite z izbiro ene od možnosti iz spustnega menija.
+- Ko je komentar podan, bo sporočen naši ekipi. Odpravili bomo težavo in vas obvestili z odgovorom na vaš komentar in zaprtjem težave.
+
+![Prikaz, kako ustvariti komentarje in označiti težave](./comment-issue.png)
```
Comment (minimalsm) [inline src/content/translations/sl/contributing/translation-program/translation-guide/index.md:41]:
```suggestion
![Prikaz, kako ustvariti komentarje in označiti težave](../../../../../contributing/translation-program/translation-guide/comment-issue.png)
```

File: src/content/translations/sl/contributing/translation-program/translation-guide/index.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+## Kaj je pomnilnik prevodov (TM)? {#translation-memory}
+
+Pomnilnik prevodov (TM) je funkcija orodja Crowdin, ki shrani vse predhodne prevedene nize na strani [ethereum.org](http://ethereum.org/). Ko je niz preveden, se samodejno shrani v TM našega projekta. To je lahko uporabno orodje, ki vam pomaga prihraniti vaš čas!
+
+- Oglejte si sekcijo "TM in MT predlogi" in lahko boste videli, kako so drugi prevajalci prevedli enak ali podoben niz. Če najdete predlog z visokim razmerjem ujemanja, ga s klikom brez težav uporabite za prevod.
+- Če na seznamu ni ničesar, lahko iščete po TM za predhodne prevode in jih ponovno uporabite za ohranjanje konsistentnosti.
+
+![Posnetek zaslona pomnilnika prevodov](./translation-memory.png)
```
Comment (minimalsm) [inline src/content/translations/sl/contributing/translation-program/translation-guide/index.md:50]:
```suggestion
![Posnetek zaslona pomnilnika prevodov](../../../../../contributing/translation-program/translation-guide/translation-memory.png)
```

File: src/content/translations/sl/contributing/translation-program/translation-guide/index.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+## Kako uporabljam Crowdin slovar? {#glossary}
+
+Ethereum terminologija je naslednji ključni del našega prevajalskega dela, saj novi tehnološki izrazi pogosto še ne bodo lokalizirani v veliko jezikov. Prav tako obstajajo izrazi, ki imajo različne pomene v različnih kontekstih. [Več o prevajanju Ethereum terminologije](#terminology)
+
+Crowdin slovar je najboljše mesto za razjasnitev izrazov in definicij. Obstajata dva načina napotitve na slovar.
+
+- Prvič, ko najdete podčrtan izraz v izvornem nizu, se lahko z miško postavite nanj in tako razkrijete njegovo definicijo.
+
+![Primer definicije iz slovarja](./glossary-definition.png)
```
Comment (minimalsm) [inline src/content/translations/sl/contributing/translation-program/translation-guide/index.md:60]:
```suggestion
![Primer definicije iz slovarja](../../../../../contributing/translation-program/translation-guide/glossary-definition.png)
```

File: src/content/translations/sl/contributing/translation-program/translation-guide/index.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+- Drugič, če najdete pojem, ki ga ne poznate, vendar ni podčrtan, lahko iščete po zavihku slovarja (tretji gumb desnega stolpca). Našli boste razlage specifičnih izrazov in tistih pogosto uporabljenih v projektu.
+
+![Posnetek zaslona, ki prikazuje, kje najti zavihek slovarja v Crowdinu](./glossary-tab.png)
```
Comment (minimalsm) [inline src/content/translations/sl/contributing/translation-program/translation-guide/index.md:64]:
```suggestion
![Posnetek zaslona, ki prikazuje, kje najti zavihek slovarja v Crowdinu](../../../../../contributing/translation-program/translation-guide/glossary-tab.png)
```

File: src/content/translations/sl/contributing/translation-program/translation-guide/index.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+
+- Če ga še vedno ne najdete, je to vaša priložnost, da dodate nov izraz! Spodbujamo vas, da ga poiščete v iskalniku in v slovar dodate opis. Bilo bo v veliko pomoč drugim prevajalcem, da bodo izraz lažje razumeli.
+
+![Posnetek zaslona, ki prikazuje kako dodati izraz v slovar v Crowdinu](./add-glossary-term.png)
```
Comment (minimalsm) [inline src/content/translations/sl/contributing/translation-program/translation-guide/index.md:68]:
```suggestion
![Posnetek zaslona, ki prikazuje kako dodati izraz v slovar v Crowdinu](../../../../../contributing/translation-program/translation-guide/add-glossary-term.png)
```

File: src/content/translations/sl/contributing/translation-program/index.md [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+   Opomba na vsebinske verzije: uporabljamo upravljanje verzij z Crowdinom, da se izognemo prevajalskim zamudam za dodane vsebine. Ko preverite jezik, na primer [Filipinščino](https://crowdin.com/project/ethereum-org/fil#), boste našli mape za vsako verzijo strani ("v2.0", "v2.1", etc.).
+
+   Spodbujamo vas, da najprej prevajate nižjo verzijo in se prebijate proti višjim verzijam (v.2.0 → v.2.1 → v.2.2 → ⋯), saj bodo nižje verzije na strani posodobljene najprej.
+
+   [Preberite si več o ethereum.org vsebinskih verzijah](/en/contributing/translation-program/content-versions/)
+
+3. **Prevajanje**  
+   Po izbiri datoteke, ki jo želite prevesti, se bo ta odprla v spletnem urejevalniku. Če niste še nikoli uporabljali Crowdina, lahko uporabite ta hitri vodnik, da pregledate osnove.
+
+   ![Crowdin spletni urejevalnik](./online-editor.png)
```
Comment (minimalsm) [inline src/content/translations/sl/contributing/translation-program/index.md:65]:
```suggestion
   ![Crowdin spletni urejevalnik](../../../../contributing/translation-program/online-editor.png)
```

File: src/content/translations/sl/contributing/translation-program/index.md [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+
+1. **[Pridružite se našemu projektu na Crowdinu](https://crowdin.com/project/ethereum-org/invite)**  
+   Morali se boste prijaviti v vaš Crowdin račun ali ga ustvariti, če ga še nimate. Vse kar je potrebno za prijavo, je le e-poštni naslov in geslo.
+
+2. **Odprite jezik, ki ga želite prevajati in poiščite dokument za delo**  
+   Vsebina spletne strani je razdeljena v številne dokumente in jezikovne verzije. Na desni lahko preverite napredek vsakega dokumenta – če je napredek prevoda pod 100 % prosimo prispevajte!
+
+   Ne najdete svojega jezika na seznamu? [Odprite težavo](https://github.com/ethereum/ethereum-org-website/issues/new/choose)
+
+   ![Prevedene in neprevedene datoteke na Crowdinu](./crowdin-files.png)
```
Comment (minimalsm) [inline src/content/translations/sl/contributing/translation-program/index.md:54]:
```suggestion
   ![Prevedene in neprevedene datoteke na Crowdinu](../../../../contributing/translation-program/crowdin-files.png)
```

File: src/content/translations/sl/nft/index.md [hunk 13]
```diff
@@ -0,0 +1,428 @@
...
+
+### ethereum.org primeri {#ethereum-org-examples}
+
+Mi NFT-je uporabljamo za vračanje našim sodelavcem in imamo tudi NFT za ime svoje domene.
+
+#### POAPi (Protokol za dokaz o prisotnosti) {#poaps}
+
+Če prispevate k ethereum.org, lahko pridobite PAOP NFT. To so zbirateljski predmeti, ki dokazujejo sodelovanje na dogodku. Nekatera kripto srečanja so že uporabila POAPe, kot obliko vstopnice za njihov dogodek. [Več o prispevanju](/en/contributing/#poap).
+
+![ethereum.org POAP](../../assets/use-cases/poap.png)
```
Comment (minimalsm) [inline src/content/translations/sl/nft/index.md:67]:
```suggestion
![ethereum.org POAP](../../../../assets/use-cases/poap.png)
```

**Review by minimalsm — CHANGES_REQUESTED** [review]

**Review by corwintines — APPROVED** [review]

**Pull request merged**